include/Rendering.h

```diff
@@ -11,6 +11,7 @@ namespace core
 	struct RenderFrameStart {};
 	struct RenderFrameEnd {};
 	struct RenderSignal {};
+	struct Render2Signal { SDL_Texture *image; uint32_t *pixels; int pitch; uint32_t w; uint32_t h; };
```

Comment:
Check if there's an alias for `SDL_Texture *`, so that we can use it. Also, all `uint32_t` should be `U32`.

Even if there's no alias, use `RawPtr<T>` instead of `T*` within dagger core.

Reply:
`*image` isn't used anywhere, should've removed it earlier.